frontend/pages/admin/ManageUsersPage/components/UsersTable/UsersTableConfig.tsx

```
@@ -7,9 +7,11 @@ import TooltipTruncatedTextCell from "components/TableContainer/DataTable/Toolti
 import TooltipWrapper from "components/TooltipWrapper";
 import PillBadge from "components/PillBadge";
 import { IInvite } from "interfaces/invite";
+import { ITeam } from "interfaces/team";
 import { IUser, UserRole } from "interfaces/user";
 import { IDropdownOption } from "interfaces/dropdownOption";
-import { generateRole, generateTeam, greyCell } from "utilities/helpers";
+import { generateRole, generateTeam, greyCell, tooltipTextWithLineBreaks } from "utilities/helpers";
```

ESLint

[error] Replace `·generateRole,·generateTeam,·greyCell,·tooltipTextWithLineBreaks·` with `⏎··generateRole,⏎··generateTeam,⏎··greyCell,⏎··tooltipTextWithLineBreaks,⏎`

(prettier/prettier)